src/bump-stdlibs.jl

```diff
@@ -130,12 +131,16 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                         git_url_markdown = _git_url_to_formatted_markdown(stdlib.git_url)
                         bumpstdlibs_sender = strip(get(ENV, "BUMPSTDLIBS_SENDER", ""))
                         bumpstdlibs_sender_ping = isempty(bumpstdlibs_sender) ? "unknown user" : "@$(bumpstdlibs_sender)"
+                        julia_version = Base.VersionNumber(Base.parsed_toml("Project.toml")["version"])
```

Comment:
Is this a typo? Presumably you should open the `VERSION` file from the Julia source tree, and use that?